.travis.yml

```diff
@@ -36,6 +36,9 @@ addons:
   hosts:
     - ads.localhost
     - iframe.localhost
+    # CURLs amp subdomain for amp-recaptcha-input,
+    # integration tests.
+    - jgla3zmib2ggq5buc4hwi5taloh6jlvzukddfr4zltz3vay5s5rq.recaptcha.localhost
```

Comment:
is this domain name fixed? pls explain a bit how this string is generated

Reply:
Yes 😄 Remember that time I showed you this URL by running it through the `amp-toolbox` curls subdomain generator thing? Running `localhost:[whatever the karma test port is]`, gives us this URL 👍

Reply:
please add that as comment